app/javascript/stylesheets/application.scss

```diff
@@ -1 +1 @@
-@import 'bootstrap'
+@import 'bootstrap';
```

Comment:
Oh, just noticed you're importing `bootstrap` here. As far as I'm aware, Bootstrap includes its own version of `normalize.css` with it, so maybe adding the stylesheet directly is not really needed? 🤔

Reply:
Oooh yeah, you're right, we don't need our normalize.css